audio_recorder/src/main.py

```diff
@@ -14,10 +14,10 @@
 
     recorder = Recorder()
     audio_properties = Audio_properties()
-    output_file_path = "../output/test.wav"
+    output_file_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", "output", "test.wav")
```

Comment:
I like that we os sensitive routines to calculate path. But, logic is still the same. We are assuming that output folder is created and that it is on one level up from this routine. this can be problematic for real deployment. 

I think it would be little bit better if we store file in "current folder".
Even better, in audio_recorder app we don't have to use file system at all. Recorded sound can be stored in stream in memory and served to main server when requested.